packages/core/types/client.d.ts

```diff
@@ -13,7 +13,7 @@ import Event from './event'
 import Session from './session'
 
 declare class Client {
-  private constructor();
+  protected constructor();
```

Comment:
I'm not sure we need this declaration at all - we don't have it on the Event public types.

Reply:
It was to deter people from instantiating the class directly but yeah… I'm not sure if we need it.